azimuth/modules/model_performance/confusion_matrix.py

```diff
@@ -31,13 +32,28 @@ def compute_on_dataset_split(self) -> List[ConfusionMatrixResponse]:  # type: ig
             ds[self.config.columns.label],
         )
         ds_mng = self.get_dataset_split_manager()
-        class_ids = list(range(ds_mng.get_num_classes()))
+        num_classes = ds_mng.get_num_classes()
+        class_ids = list(range(num_classes))
         cf = confusion_matrix(
             y_true=labels,
             y_pred=predictions,
             labels=class_ids,
             normalize="true" if self.mod_options.cf_normalized else None,
         )
+
+        class_names = ds_mng.get_class_names()
+
+        # Put the rejection class last for the confusion matrix
+        rejection_idx = ds_mng.rejection_class_idx
+        if rejection_idx != max(class_ids):
```

Comment:
I am not too keen on the `max()` thing, as it is O(n) (looping through the whole list), whereas `num_classes - 1` is O(1) (constant). Again, it is unlikely that `class_ids` will get big enough for that to make a difference, but sub-optimal code always looks wrong to me.
```suggestion
        if rejection_idx != num_classes - 1:
```